tutorials/getstarted.jl

```diff
@@ -0,0 +1,1225 @@
+
+The first one concerns vectors or matrices of data on a manifold, the [PowerManifold](https://juliamanifolds.github.io/ManifoldsBase.jl/stable/manifolds.html#sec-power-manifold).
+"""
+
+# ╔═╡ 114b46c3-654d-4b1c-b8a9-3acc5939a25e
+M₄ = M₂^2
+
+# ╔═╡ 1c7deff8-f9bf-4e11-9c40-47a6970ee502
+md"Then points are represented by arrays, where the power manifold dimension is added in the end. In other words – for the hyperbolic manifold here, we have a matrix with 2 columns, where each column is a valid point on hyperbolic space."
+
+# ╔═╡ 741ceb0c-f5e8-438e-b3f1-857cae2d451d
+p = [0 0; 0 1; 1 sqrt(2)]
+
+# ╔═╡ 59a99512-cb9c-4106-ad1b-69bf947d604b
+[is_point(M₂, p[:, 1]), is_point(M₂, p[:, 2])]
+
+# ╔═╡ ff62cb96-d1ae-4fed-9101-e728d9f2fe18
+md"But of course the method we used previously also works for power manifolds:"
+
+# ╔═╡ 81e0cc2b-1e79-411c-aee1-d1761d0b95d2
+is_point(M₄, p)
+
+# ╔═╡ 8e85742c-fa06-4212-bace-81479b31d9e9
+md"Note that nested power manifolds are combined into one as in"
+
+# ╔═╡ 262f0ca4-4ef5-4670-a3d2-c86a74884d97
+M₄₂ = M₄^4
+
+# ╔═╡ 0c46f29e-126b-42c0-a96c-24f369c5fd80
+md"which represents ``2\times 4`` – matrices of hyperbolic points represented in ``3\times 2\times 4`` arrays."
+
+# ╔═╡ 0d56dddc-0f52-48e1-b81f-bc08fbcdfddf
+md"We can – alternatively – use a power manifold with nested arrays"
+
+# ╔═╡ 01cae5a1-0d0d-4163-8713-52efb79d5043
+M₅ = PowerManifold(M₃, NestedPowerRepresentation(), 2)
+
+# ╔═╡ 33bab279-4258-4402-a4e5-da6fec1a88bf
+md"which emphasizes that we have vectors of length 2 that contain points, so we store them that way."
+
+# ╔═╡ e20ec2bc-7488-4d09-9a3a-17c06d6a4bfa
+p₂ = [[0.0, 0.0, 1.0], [0.0, 1.0, 0.0]]
+
+# ╔═╡ 58eb08aa-1963-4b01-bbee-48ca9ae68e9e
+md"Top unify both representations, elements of the power manifold can also be accessed in the classical indexing fashion, if we start with the corresponding manifold first. This way one can implement algorithms also independent of which representation is used."
+
+# ╔═╡ bd676b86-060d-4e94-af01-fa1fe88a4cc7
+p[M₄, 1]
+
+# ╔═╡ 44840536-f75e-4db9-ad70-ded60beac987
+p₂[M₅, 2]
+
+# ╔═╡ 5570facf-5d04-42fc-bc40-28acad1fbc40
+md"""
+Another construtor is the [ProductManifold](https://juliamanifolds.github.io/Manifolds.jl/previews/PR534/manifolds/product.html) to combine different manifolds. Here of course the order matters.
+First we construct these using ``×``
+"""
+
+# ╔═╡ a02f085b-eaef-4a8f-a8fe-e306e956dd2c
+M₆ = M₂ × M₃
+
+# ╔═╡ 0f135d94-25c8-499d-a9e0-169db76cb901
+md"Since now the representations might differ from element to element, we have to encapsulate these in their own type."
+
+# ╔═╡ fa0baedb-636e-4ac8-9779-039625ca8267
+p₃ = Manifolds.ArrayPartition([0, 0, 1], [0, 1, 0])
+
+# ╔═╡ 36140614-7d40-4281-ba75-2b7b7106e637
+md"""
+Here `ArrayPartition` taken from [`RecursiveTools.jl`](https://github.com/SciML/RecursiveArrayTools.jl) to store the point on the product manifold efficiently in one array, still allowing efficient acces to the product elements.
```

Comment:
```suggestion
Here `ArrayPartition` taken from [`RecursiveArrayTools.jl`](https://github.com/SciML/RecursiveArrayTools.jl) to store the point on the product manifold efficiently in one array, still allowing efficient access to the product elements.
```

Reply:
Oh, yeah, typing and thinking in parallel was difficult here.